examples/constructor/src/call.rs

```diff
@@ -246,18 +246,20 @@ mod wasm {
             Some((left == right).encode())
         }
 
-        fn if_else(self, previous: Option<CallResult>) -> Option<Vec<u8>> {
-            let Self::IfElse(flag, true_call, false_call) = self else {
+        fn if_else(self, mut previous: Option<CallResult>) -> Option<Vec<u8>> {
```

Comment:
```suggestion
        fn if_else(self, mut previous: Option<CallResult>) -> Option<Vec<Vec<u8>>> {
```

shouldn't this result be `Vec<Vec<u8>>`, or we will only have the result of the last call

Reply:
nope, we don't need all of the return values